docs/MatrixOne/Develop/schema-design/1.1-vector.md

```diff
@@ -0,0 +1,143 @@
+# Vectors
+
+## What Are Vectors
+
+In a database, vectors are typically sets of numbers arranged in a specific way to represent some form of data or features. These vectors can be one-dimensional, multidimensional, or data structures with higher dimensions. In the fields of machine learning and data analysis, vectors are used to represent data points, features, or model parameters.
```

Comment:
needs change

Reply:
I reedited: 
In a database, vectors typically consist of organized sets of numbers arranged in a specific manner to represent various data or characteristics. These vectors can exist in one-dimensional, multidimensional, or even higher-dimensional data structures. In machine learning and data analysis, vectors represent data points, features, or model parameters.